lib/in/create-package-summary.js

```diff
@@ -30,6 +31,15 @@ function createPackageSummary(moduleName, currentState) {
         return false;
     }
 
+    // Ignore specified '--ignore' package globs
+    const ignore = currentState.get('ignore');
+    if (ignore) {
+        const ignoreMatch = Array.isArray(ignore) ? ignore.some(ignoredModule => minimatch(moduleName, ignoredModule)) : minimatch(moduleName, ignore);
```

Comment:
@lijunle Seems that @eldinoyev was right, depcheck used only for checking unused dependencies but here we want to ignore package from all checks.